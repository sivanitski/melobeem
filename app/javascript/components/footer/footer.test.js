import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";

import Footer from "./footer";

Enzyme.configure({
  adapter: new Adapter(),
});

describe("Footer component", () => {
  it("should render Footer component", () => {
    const component = renderer
      .create(
        <Router>
          <Footer />
        </Router>
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});
