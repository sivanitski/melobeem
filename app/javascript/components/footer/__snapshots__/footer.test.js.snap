// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer component 1`] = `
<div
  className="footer"
>
  <a
    className="footer__item"
    href="/"
    onClick={[Function]}
  >
    <test-file-stub />
    <div
      className="footer__item__title"
    >
      Leaderboard
    </div>
  </a>
  <a
    className="footer__item"
  >
    <test-file-stub />
    <div
      className="footer__item__title"
    >
      Spinner
    </div>
  </a>
  <a
    className="footer__button"
  >
    <test-file-stub />
  </a>
  <a
    className="footer__item"
  >
    <test-file-stub />
    <div
      className="footer__item__title"
    >
      Levels
    </div>
  </a>
  <a
    className="footer__item"
  >
    <test-file-stub />
    <div
      className="footer__item__title"
    >
      Profile
    </div>
  </a>
</div>
`;
